File: project.javafol/src/main/BookmyTrip/datahandler/Booking.java
package main.BookmyTrip.datahandler;

import java.io.*;
import java.util.*;
import java.time.*;
import java.time.LocalDate;

import main.BookmyTrip.datahandler.Customer;

public class Booking {
    public long bookingID;
    public long custID;
    public long mills = System.currentTimeMillis();
    public LocalDate dateofbooking;
    public LocalTime timeofbooking;
    public final long x= 99999;
    public String travelMode;
    public float busFare;
    public float trainFare;
    public float flightFare;
    public float totalFare;
    public float tripDist;
    public String depDest;
    public String arrDest;
    public String depDate;
    public String descision;
    public int hotelChoice;
    public String hotelName;
    public float hotelCharge;
    public int paymentMode;
    public String cardNum;
    public String cardExpiry;
    int ccv;
    int otp;
    String chequeNum;
    boolean bookingStatus;
    
    public Booking() {
    };


}

File: project.pythfol/Booking.py
import csv
import time
from datetime import date

class Booking :
    bookingID = 0
    dateofbooking = date.today()
    timeofbooking = time.localtime()
    x = 999999
    travelMode = "null"
    busFare = 0.0
    trainFare = 0.0
    flightFare = 0.0
    fareTotal = 0.0
    tripDist = 0.0
    depDest = "null"
    arrDest = "null"
    depDate = date